ngraph_bridge/ie_executable.cc

```diff
@@ -44,6 +44,11 @@ IE_Executable::IE_Executable(shared_ptr<Function> func, string device)
   m_network = InferenceEngine::CNNNetwork(func);
   set_parameters_and_results(*func);
 
+  if (std::getenv("NGRAPH_TF_DUMP_GRAPHS")) {
```

Comment:
May be we can use another env var for this? NGRAPH_TF_DUMP_GRAPHS dumps a lot of graphs and you may be interested only in IE graphs.

Reply:
As this flag is used for debugging, more graphs might not be a big issue.

Reply:
Yes, too many undocumented env vars get a little unwieldy so I decided to reuse `NGRAPH_TF_DUMP_GRAPHS`.